dependabot/resources/extensions.json

```diff
@@ -902,7 +902,7 @@
             "default_branch": "main",
             "auto_merge": true,
             "push_to_central": true,
-            "is_extended_library_module": true,
+            "is_extended_library_module": false,
```

Comment:
This is an autogenerated file. This will be overriden in next build run. Add this flag to module_list.json.

Reply:
Updated.